storage_service/locations/models/package.py

```diff
@@ -2320,7 +2330,7 @@ def finish_reingest(
                         ' "{}"; does not match any of the following recognized'
                         " options: {}".format(
                             compression_algorithm,
-                            ", ".join(utils.COMPRESSION_ALGORITHMS),
+                            ", ".join(compression_options)
```

Comment:
Nice catch! You could also switch this logger to use lazy %-style logging, but not a must-do.